### DISCUSSION TITLE
How to enable hostNameVerifier on server side in http4s BlazeServerBuilder

### DISCUSSION THREAD

**Discussion by tpsaitwal** [http4s/http4s#6827] [Q&A]
In one of my project we are moving away from **akka** (v10.2.9) to **http4s** (v0.23.12). In akka we are creating **http** **server** using **akka.http.scaladsl.Http** object which internally creates **HttpConnectionContext** for server using **AkkaSSLConfig** which by default has **hostNameVerifier** enabled on **server** side as well, Which checks host names against CN and SAN. You can disable this **hostNameVerification** using this parameter
 `akka.ssl-config.loose.disableHostnameVerification = true`

When I dug deeper into this I got to know **hostNameVerification** should enabled on **Client** side only to avoid **man in the middle attack**.

However, while moving from **akka** to **http4s** I still want to keep the functionality of **hostNameVerification**. I read the **http4s** [documentation](https://http4s.org/v0.22/docs/service.html) and I am using **BlazeServerBuilder** but I didn't find any provision to enable **hostNameVerification** on **server** side. How can this be achieved with **http4s** and **scala**.

**Comment by armanbilge** [comment]
Blaze questions can be asked here:
- https://github.com/http4s/blaze
